Assets/HoloToolkit/SpatialMapping/Scripts/TapToPlace.cs

```diff
@@ -38,7 +45,13 @@ public class TapToPlace : MonoBehaviour, IInputClickHandler
         /// </summary>
         private bool placing;
```

Comment:
I could see that being useful. For instance if you want to automatically place an object in a location when the object "hovers" over that location. I agree with making it public.